### ISSUE TITLE
compile error on master

### ISSUE THREAD

**Issue by mabondarenko** [pboettch/json-schema-validator#38]
Hi! I'm trying to compile with my project, but get an compile error. That solved by added #include <sstream> in json-uri.cpp:

```
[  1%] Building CXX object contrib/json-schema-validator/CMakeFiles/json-schema-validator.dir/src/json-uri.cpp.o
/home/max/src/test/contrib/json-schema-validator/src/json-uri.cpp: In member function ‘const string nlohmann::json_uri::url() const’:
/home/max/src/test/contrib/json-schema-validator/src/json-uri.cpp:101:21: error: aggregate ‘std::stringstream s’ has incomplete type and cannot be defined
   std::stringstream s;
                     ^
/home/max/src/test/contrib/json-schema-validator/src/json-uri.cpp: In member function ‘std::__cxx11::string nlohmann::json_uri::to_string() const’:
/home/max/src/test/contrib/json-schema-validator/src/json-uri.cpp:114:21: error: aggregate ‘std::stringstream s’ has incomplete type and cannot be defined
   std::stringstream s;
                     ^
```

**Comment by pboettch** [comment]
Thanks for reporting. Are you able to issue an pull-request to fix it?

**Comment by mabondarenko** [comment]
> Thanks for reporting. Are you able to issue an pull-request to fix it?

done: https://github.com/pboettch/json-schema-validator/pull/39

**Comment by pboettch** [comment]
fixed via #39
